app/frontend/src/components/Navigation/Navigation.tsx

```diff
@@ -91,7 +92,13 @@ const useStyles = makeStyles((theme) => ({
     paddingRight: 0,
   },
   nav: {
+    alignItems: "center",
     display: "flex",
+    flex: 0,
```

Comment:
Firefox makes the left nav bar wider than it needs to be without this. Again not sure why other browsers don't do this. Luckily with this combo, it works on all of them now. The fun of dealing with inconsistencies in the frontend...